spec/api/search_spec.rb

```diff
@@ -1,33 +1,40 @@
 require 'spec_helper'
 require 'unicode_shared_examples'
 
-describe "app" do
-  describe "search" do
+describe 'app' do
+  describe 'search' do
     include_context 'search_enabled'
 
     before (:each) { set_api_key_header }
-    let(:author) { create_test_user(42) }
-    let(:course_id) { "test/course/id" }
+    let(:author) { create(:user) }
+    let(:course_id) { 'test/course/id' }
 
     def get_result_ids(result)
-      result["collection"].map { |t| t["id"] }
+      result['collection'].map { |t| t['id'] }
     end
 
     describe "GET /api/v1/search/threads" do
```

Comment:
We should try to make quotes consistent.  Double-quotes are simplest, but not sure which is used for most of these lines?